packages/client/components/EventTypeFilterMenuItemLabel.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import {timelineEventTypeMenuIcons, timelineEventTypeMenuLabels} from '../utils/constants'
+import TimelineEventTypeIcon from './TimelineEventTypeIcon'
+import MenuItemLabel from './MenuItemLabel'
+import {FilterLabels} from '../types/constEnums'
+import {TimelineEventEnum} from '../__generated__/MyDashboardTimelineQuery.graphql'
+import CardsSVG from './CardsSVG'
+
+interface Props {
+  eventType?: TimelineEventEnum
+}
+
+const EventTypeFilterMenuItemLabel = (props: Props) => {
+  const {eventType} = props
+  const eventTypeLabel = eventType
+    ? timelineEventTypeMenuLabels[eventType]
+    : FilterLabels.ALL_EVENTS
+  const eventTypeIconName = eventType ? timelineEventTypeMenuIcons[eventType] : 'timeline'
+  return (
+    <>
+      <span className='py-1 pl-4'>
+        <TimelineEventTypeIcon iconName={eventTypeIconName} />
+        {!eventTypeIconName && <CardsSVG />}
```

Comment:
+1 this feels a bit odd. So basically all `eventType`s that aren't in the `timelineEventTypeMenuIcons` map get the `CardSVG` logo. Might be technically correct, still feels a bit odd as logic without a comment.